src/commands/api-mesh/source/install.js

```diff
@@ -0,0 +1,219 @@
+		let variables = flags.variable
+			? flags.variable.reduce((obj, val) => {
+					const splited = val.split('=');
+					obj[splited[0].trim()] = splited[1].trim();
+					return obj;
+			  }, {})
+			: {};
+		if (filepath) {
+			try {
+				variables = { ...variables, ...JSON.parse(await readFile(filepath, 'utf8')) };
+			} catch (e) {
+				this.error(`Something went wrong when reading variables file.` + `\n${e.message}`);
+			}
+		}
+
+		let meshId = null;
+		if (!args.source && !flags.source) {
+			this.error(
+				`The "aio api-mesh:source:install" command requires additional parameters` +
+					`\nUse "aio api-mesh:source:install --help" to see parameters information.`,
+			);
+		}
+		let list;
+		try {
+			list = await this.sourceRegistryStorage.getList();
+		} catch (err) {
+			this.error(`Cannot get the list of sources: ${err}`);
+		}
+		const sources = flags.source ? flags.source : [args.source];
+		const sourceConfigs = [];
+		for (const source of sources) {
+			let [name, version] = source.split('@');
+			const normalizedName = this.normalizeName(name);
+			if (!list[normalizedName]) {
+				this.error(
+					chalk.red(
+						`The source with the name "${name}" doesn't exist.` +
+							`\nUse "aio api-mesh:source:discover" command to see avaliable sources.`,
+					),
+				);
+			}
+			version = version || list[normalizedName].latest;
+			if (!list[normalizedName].versions.includes(version)) {
+				this.error(
+					chalk.red(
+						`The version "${version}" for source name "${name}" doesn't exist.` +
+							`\nUse "aio api-mesh:source:discover" command to see avaliable source versions.`,
+					),
+				);
+			}
+			const sourceConfig = await this.sourceRegistryStorage.get(name, version);
+			const jsonInterpolate = new JsonInterpolate({ variablesSchema: sourceConfig.variables });
+			const sourceProviderString = JSON.stringify(sourceConfig.provider);
+			const sourceVariables = jsonInterpolate.getJsonVariables(sourceProviderString);
+			const missedVariables = jsonInterpolate.getMissedVariables(variables, sourceVariables);
+			for (const missedVariable of missedVariables) {
+				variables[missedVariable.name] = await promptInput(
+					`Enter the value for variable ${missedVariable.name}:`,
+				);
+			}
+
+			const { error, data } = jsonInterpolate.interpolate(
+				JSON.stringify(sourceConfig.provider),
+				variables,
+			);
+			if (error) {
+				this.error(chalk.red(`${error.message}\n${error.list.map(err => err.message).join('\n')}`));
+			}
+			sourceConfigs.push(JSON.parse(data));
+		}
+
+		try {
+			meshId = await getMeshId(imsOrgId, projectId, workspaceId);
+		} catch (err) {
+			this.error(
+				`Unable to get mesh ID. Please check the details and try again. RequestId: ${global.requestId}`,
+			);
+		}
+
+		if (!meshId) {
+			this.error(
+				`Unable to get mesh config. No mesh found for Org(${imsOrgId}) -> Project(${projectId}) -> Workspace(${workspaceId}). Please check the details and try again.`,
```

Comment:
```suggestion
				`Unable to get the mesh config. No configured mesh found for Org(${imsOrgId}) -> Project(${projectId}) -> Workspace(${workspaceId}). Please check the details and try again.`,
```

Reply:
This error message is copy of message from other API mesh commands. I prefer to have consistent message

Reply:
@revanth0212 I believe @VladimirZaets is concerned that the error messages on lines 122, 131, 157 and 172 all use inconsistent terminology when referring to the mesh. (e.g. what is the mesh, vs the mesh configuration, vs the mesh configuration file, etc) Additionally, some of the error messages appear to be designed to communicate the same information, but are worded differently.